source/Patches/EndGameSummary.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using UnityEngine;
+using TownOfUs.Roles;
+using TownOfUs.Roles.Modifiers;
+using HarmonyLib;
+
+namespace TownOfUs.Patches
+{
+    [HarmonyPatch]
+    static class EndGameSummary
+    {
+        static class AdditionalTempData
+        {
+            public static List<PlayerInfo> playerData = new List<PlayerInfo>();
```

Comment:
Nit: can be `readonly` right?

Reply:
not possible because it gets assigned to on line 71.